fcrepo-kernel-modeshape/src/main/java/org/fcrepo/kernel/modeshape/FedoraResourceImpl.java

```diff
@@ -457,8 +457,14 @@ private long getTimestamp(final String property, final long created) throws Repo
         LOGGER.trace("Using {} date", property);
         final long timestamp = getProperty(property).getDate().getTimeInMillis();
         if (timestamp < created && created > NO_TIME && !isFrozenResource()) {
-            LOGGER.trace("Updating {} with later created date", property);
-            getNode().setProperty(property, created);
+            try {
+                LOGGER.trace("Updating {} with later created date", property);
+                getNode().setProperty(property, created);
+            } catch (final javax.jcr.AccessDeniedException e) {
+                LOGGER.trace("Unable to update {} with the later created date", property);
+            } catch (final javax.jcr.RepositoryException e) {
+                LOGGER.warn("Unable to update " + property + " with the later created date", e);
```

Comment:
I don't know how or whether a RepositoryException would be thrown here... until I added this line, I got a 409 from an integration test (though I couldnt' reproduce it).  Since I characterize this secret update operation within our method to get a timestamp as optional, I figured any exception thrown can be eaten and logged.

It's important to include the stack trace in the log in case this ever actually happens.  When these issues arise in production servers they are often hard to reproduce, but if we include the JCR exception as the root cause of the exception that actually gets logged, there's a chance we'll have the stack trace in our logs and we can see excactly what section of code to investigate as the cause of our unexpected error.